pkg/sysinfo/sysinfo.go

```diff
@@ -45,8 +46,13 @@ func New(quiet bool) *SysInfo {
 			logrus.Warnf("%v", err)
 		}
 	} else {
-		_, err1 := ioutil.ReadFile(path.Join(cgroupCpuMountpoint, "cpu.cfs_quota_us"))
-		sysInfo.CpuCfsQuota = err1 == nil
+		_, err := ioutil.ReadFile(path.Join(cgroupCpuMountpoint, "cpu.cfs_period_us"))
+		sysInfo.CpuCfsPeriod = err == nil
+		if !sysInfo.CpuCfsPeriod && !quiet {
+			logrus.Warnf("WARING: Your kernel does not support cgroup cfs period")
```

Comment:
`logrus.Warf` will has a `WARN`, I think this is no need `WARNING` in your message